pkg/util/mount/mount.go

```diff
@@ -49,3 +49,30 @@ type MountPoint struct {
 	Freq   int
 	Pass   int
 }
+
+// Examines /proc/mounts to find all other references to the device referenced
```

Comment:
@rootfs things are in the files they are to ensure that the build flags for
each function / type didn't change across this commit.
On Wed, Mar 4, 2015 at 7:58 PM Huamin Chen <notifications@github.com> wrote:

> In pkg/util/mount/mount.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5036#discussion_r25830927>
> :
>
> > @@ -49,3 +49,30 @@ type MountPoint struct {
> >  	Freq   int
> >  	Pass   int
> >  }
> > +
> > +// Examines /proc/mounts to find all other references to the device referenced
>
> why not just create a mount_util.go?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5036/files#r25830927>
> .
>